src/mlpack/tests/main_tests/range_search_utils.hpp

```diff
@@ -18,15 +18,18 @@
 #include <mlpack/core/util/mlpack_main.hpp>
 
 /**
- * Convert a model to a string using the text_oarchive of boost::serialization.
+ * Convert a model to a string using the text_oarchive of cereal.
  *
  * @param model RSModel to be converted to string.
  */
 inline std::string ModelToString(RSModel* model)
 {
   std::ostringstream oss;
-  boost::archive::text_oarchive oa(oss);
-  oa << model;
+  cereal::JSONOutputArchive oa(oss);
+  // It is not possible to serialize this model, I am getting an error
+  // of not serialize function is found fir this model, even that we have
+  // declared that in the rs_model_impl
+  oa << CEREAL_POINTER(model);
```

Comment:
```suggestion
  oa(CEREAL_POINTER(model));
```
Is this comment still accurate, also?  I think it may need to be removed, since this appears to work now (correct me if I am wrong).

Reply:
This error needs to be removed you are right